worker/lib/workers/qemu.ts

```diff
@@ -234,8 +237,37 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 		});
 	}
 
-	private async iptablesRules(){
-		await execProm(`iptables-legacy -t nat -A POSTROUTING ! -o ${this.qemuOptions.network.bridgeName} --source ${this.qemuOptions.network.bridgeAddress}/24 -j MASQUERADE -m comment --comment ${this.iptablesComment}`);
+	private async iptablesRules(bridgeName: string, bridgeAddress: string){
+		this.iptablesComment = `teardown_${bridgeName}`
+		await execProm(`iptables-legacy -t nat -A POSTROUTING ! -o ${bridgeName} --source ${bridgeAddress}/24 -j MASQUERADE -m comment --comment ${this.iptablesComment}`);
+	}
+
+	private async getBridgeIpAddress(){
+		const interfaces = JSON.parse((await execProm(`ip -json a`)).stdout);
+		let subnet = 100;
+		while(subnet < 255){
+			let subenetAddr = `192.168.${subnet}.1`;
```

Comment:
Can we expose the network as an environment variable with CIDR?

The default could be something like `10.10.0.0/16` where the CIDR must be between 8-20? (check my math)

Reply:
After discussing this let's avoid the initial complexity and just change the port range to `10.10.x.x` for now.